add_to_db.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require 'functions.php';
+$db = get_db();
+$winner = $_POST;
+test_input();
+add_winner($db, $winner);
+/*
+* Inserts new entry into the DB
+*
+* @param object DB of previous winners
+* @param validated array
+*
+* @return
+*/
+function add_winner(object $db, array $winner)
```

Comment:
just FYI - you can return type hint any functions that have no return value to void

`function add_winner(object $db, array $winner) : void`

No need to actually set this though if you don't want to!